test/gdb_script.py

```diff
@@ -96,11 +104,16 @@
 }
 
 
+def gdb_execute(cmd):
+    print("Executing command:", cmd)
```

Comment:
Was this intentionally left in or is it just for debugging?

Reply:
Intentional. It's helpful to have the gdb commands listed in the `output.txt` file.